src/SignalR/clients/csharp/Http.Connections.Client/src/Microsoft.AspNetCore.Http.Connections.Client.csproj

```diff
@@ -4,6 +4,7 @@
     <Description>Client for ASP.NET Core Connection Handlers</Description>
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <IsShippingPackage>true</IsShippingPackage>
+    <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
```

Comment:
Is this only needed because this is targeting the blazor-wasm branch? Or will the csproj changes be merged back to master?

Reply:
The former, in master & 3.1 we won't need the explicit `packageReference` below. If it was just a `Reference` in this branch it'd resolve as a projectReference to a project that isn't building